Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab

```diff
@@ -826,3 +831,211 @@ MeshRenderer:
   m_SortingLayerID: 0
   m_SortingLayer: 0
   m_SortingOrder: 0
+--- !u!1 &7299770270808978802
+GameObject:
```

Comment:
Is this new GameObject intentional?

Reply:
Yes, it was missing the text label. Added it back to make it consistent with HoloLens 2 buttons. 
I'll check the voice command string!